app/api/v1/redflags/models.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+from flask import request
+from flask_restful import reqparse
+
+"""This module handles incidence data model"""
+
+db = [] 
+
+def validator(value):
+    if not value:
+        raise ValueError("Must not be an empty string")
+        
+parser = reqparse.RequestParser(bundle_errors=True)
+
+parser.add_argument('createdBy',
+                    type=str,
+                    required=True,
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('location',
+                    type=str,
+                    required=True,
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('images',
+                    action='append',
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('videos',
+                    action='append',
+                    help="Please fill this field"
+                    )
+
+parser.add_argument('comment',
+                    type=str,
+                    required=True,
+                    help="Please fill in this field"
+                    )
+
+class Incidence:
+    def __init__(self, createdBy, location,image, video, comment, incidence_type): 
+        """creates instance variables for the Incidence class"""
+        self.counter = 0
+        self.status = 'draft'
+        self.createdOn = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        self.image = image
+        self.video = video
+        self.createdBy = createdBy
+        self.location = location
+        self.comment = comment
+        self.incidence_type = incidence_type
+
+    def get_all_incidences(self):
+        return db
+    
+    def create_incidence(self):
+        args = parser.parse_args()
+        data = {}
```

Comment:
just assign this directly to self, then you can do away with the assignments below. 
```suggestion
        data = self
```